src/App.js

```diff
@@ -47,5 +50,32 @@ export default function App() {
     )
     app.append(errorMessage)
   }
+
+  function onNavigate(text) {
+    if (text === 'Home') {
+      console.log('Home')
+      // homePage.show()
```

Comment:
If you don't need/use code then it would be best to get rid of the comments before pushing to main.